assets/css/stylesheet.css

```diff
@@ -185,6 +191,22 @@ header > h1 {
 	color: #fff;
 }
 
+.middle
+{
+	width:-moz-fit-content;
+	width: -webkit-fit-content;
+	width: fit-content;
+	margin: auto;
+}
+
+.col-sm-5
+{
+	margin: auto;
+}
+.hdn
```

Comment:
Je n'aime vraiment pas mélanger fond et forme, et cette solution me parraît sale. Qu'on ne me dise pas qu'il n'y a pas de solution propre avec des [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/):

```css
.card {
	display:flex;
}
.cards-row {
	justify-content: center;
}
```

Reply:
Oups, j'ai oublié d'enlever ces règles, j'ai changé de technique, elles ne sont plus utiles ni utilisées

Reply:
Y compris le `.hdn` ? `flex` me semble vraiment idéal dans ce cas-là.

Reply:
Non me .hdn est utilisé.
Bootstrap est déjà en flex mais la taille des colonnes dépend du nombre de colonnes. Les 2 cartes serait trop larges c'est pour ça que j'en ai rajouté une cachée